README.md

```diff
@@ -67,6 +68,19 @@ either:
 * `git clone --recursive https://github.com/equinor/dlisio`
 * `git submodule update --init external/fmt`
 
+layered-file-protocols has to be installed from source if you don't already
+have it on your system:
```

Comment:
Once again: why don't we want to release lfp first and do not have to fail dlisio l if we screw something up in lfp master?

Reply:
It's probably a good idea to not tie dlisio too tightly to the lfp version, and to be able to compile against several releases and distributions. If things fails because someone links against a buggy lfp, then that's a problem for them at that time _when building from source_.

When providing a complete release, the story is different, but at that point we control the full bundle anyway, so it's not an issue.